src/Plugin/os2web/DataLookup/ServiceplatformenCPRExtended.php

```diff
@@ -122,13 +138,20 @@ public function lookup($cpr, $allowCprTestModeReplace = TRUE) {
           $relationship->barn = [$relationship->barn];
         }
 
-        foreach ($relationship->barn as $child) {
-          $childCprResult = $this->lookup($child->personnummer, FALSE);
-
-          $children[] = [
-            'cpr' => $childCprResult->getCpr(),
-            'name' => $childCprResult->getName(),
+        foreach ($relationship->barn as $relationshipChild) {
+          // Sometimes CPR lookup can return no results, creating child without
+          // name.
```

Comment:
This sounds strange to me. 

Is the idea to simply return the information we already know even though the CPR lookup fails?

Reply:
I experienced that with some children CPR-s. Thera was a case with one of the test CPR-s, it returned a child CPR that could not be looked up (probably wasn't added to the list of available test CPR-s).

So my idea here is to show the information we have (a child CPR number in this case, even though it cannot be looked up), instead of showing nothing.

That gives you a way of addressing a problem further.

Reply:
Good idea!